doc/nrf/ug_thread_power_consumption.rst

```diff
@@ -0,0 +1,102 @@
+.. _thread_power_consumption:
+
+OpenThread power consumption
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page provides information about the amount of power used by Sleep End Devices (SEDs) and Synchronized Sleep End Devices (SSEDs) using OpenThread stack.
+
+.. _thread_power_consumption_methodology:
+
+Measurements methodology
+************************
+
+The measurement setup consists of:
+
+* An nrf52840dk_nrf52840 board used as Thread leader.
+* A DUT board used as Thread child.
+* A `Power Profiler Kit II (PPK2)`_ attached to the DUT according to its "Quick start" guide.
+
+The leader board is flashed with the regular :ref:`Thread CLI sample <ot_cli_sample>` firmware.
+The DUT board is flashed with the :ref:`Thread CLI sample low power mode <ot_cli_sample_low_power>` firmware.
+
+After the Thread network is enabled on the leader, the child is configured with the desired parameters.
+Once attached to the network the child device will switch to sleep mode (disabling all peripherals) and wake up according to the configured period only.
+For the SED case, it will regularly send MAC Data Poll Commands.
+For the SSED case, it will regularly switch to radio receive mode.
+The PPK2 is used to collect power consumption data during at least 60 seconds.
+Afterwards, the CSV data is parsed with a script to produce the below results.
+
+.. _thread_power_consumption_data:
+
+Power consumption data
+**********************
+
+.. tabs::
```

Comment:
I think this would work as only one tabs contruct with a "Thread 1.3 SED" tab and a "Thread 1.3 SSED" tab, each containing the respective configuration table and power consumption table.

Reply:
Yes, it's as intended: https://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-10030/nrf/ug_thread_power_consumption.html#power-consumption-data

I've removed the tables' descriptions since they are repetitive with the tab's title.

Reply:
It does have some unintended behavior with being able to change the lower set of tabs (Power consumption) without the top ones changing. Using just the top one seems to change both sets.